Tests/TestingApps/PurchaseTesterSwiftUI/Shared/ReceiptInspector.swift

```diff
@@ -13,83 +14,166 @@ import ReceiptParser
 struct ReceiptInspectorView: View {
 
     @State private var encodedReceipt: String = ""
-    @State private var parsedReceipt: String = ""
-    @State private var verifyReceiptResult: String = ""
+    @State private var parsedReceipt: String = "A JSON version of the parsed receipt will show up here."
+    @State private var verifyReceiptResult: String = "Results of calling /verifyReceipt will show up here."
     @State private var sharedSecret: String = ""
+    
+    @State private var sk2AuthToken: String = ""
+    @State private var sk2AppConfigID: String = ""
+    @State private var sk2TransactionID: String = ""
+    @State private var sk2ReceiptResult: String = "Results of calling StoreKit 2 diagnostics will show up here."
 
     var body: some View {
         VStack {
-            Text("Receipt Parser")
```

Comment:
extracted the common things in these to make them easier to reuse and clean up the view a bit